setup.py

```diff
@@ -19,8 +19,9 @@
 hive_sensor = ["hmsclient>=0.0.1,<1.0.0"]
 notebook = ["papermill>=1.2.0", "nbconvert>=6.0.7", "ipykernel>=5.0.0"]
 sagemaker = ["sagemaker-training>=3.6.2,<4.0.0"]
+mysql = ["sqlalchemy>=1.4.7", "pymysql>=1.0.2"]
```

Comment:
let's nix this line?  These are all for the old style of plugins (the ones that live in flytekit/plugins rather than the new top level plugins folder).  We're trying to deprecate all the old-api related things.